### PR TITLE
displays stores associated with reviews on the profile tab 

### PR DESCRIPTION
**Pull request by TylerHernandez**
while displaying email associated with reviews on the actual coffeeshop tab

### CODE REVIEW

File: src/main/java/com/google/sps/data/RecommendationEngine.java [hunk 1]
```diff
@@ -9,7 +9,10 @@
 public class RecommendationEngine {
 
   public static Optional<Drink> getDrink(String storeId, String drinkName, List<Drink> drinkList) {
```
Comment (mutisog) [inline src/main/java/com/google/sps/data/RecommendationEngine.java:11]: Add a brief comment outlining what getDrink returns.

File: src/main/java/com/google/sps/servlets/RecommendServlet.java [hunk 2]
```diff
@@ -37,10 +37,25 @@
...
+    // adds all drinks from the storeid's we want into a list
+    for (String storeId : listStoreIds) {
+      for (Drink drink : DrinkDAO.getDrinksByStore(storeId)) {
+        drinkList.add(drink);
+      }
+    }
+
+    // puts the beverage into a hashmap which maps to its bestshop storeid
+    for (String beverage : beverages) {
+      beverage = beverage.trim();
```
Comment (mutisog) [inline src/main/java/com/google/sps/servlets/RecommendServlet.java:56]: Good job on trimming the leading and trailing whitespace from the drink string 👍

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -55,16 +55,11 @@ function createListElementUser(drink) {
   ratingElement.className = 'drink';
 
   const drinkElement = document.createElement('span');
-  var store = idToStore(drink.store)
-  drinkElement.innerText = store +", "+drink.drink + ", " + drink.rating + "/5, " + drink.content;
+  drinkElement.innerText = drink.storeInfo +", "+drink.drink + ", " + drink.rating + "/5, " + drink.content;
```
Comment (mutisog) [inline src/main/webapp/script.js:58]: Please attach a couple screenshots to show how the comment is rendered on the TYD Coffee UI.

**Review by mutisog — APPROVED** [review]
Couple of comments, but otherwise LGTM.

File: src/test/java/com/google/sps/data/CommentDAOTest.java [hunk 4]
```diff
@@ -55,7 +55,8 @@ public void tearDown() {
 
   @Test
   public void testGetCommentByID() {
-    Comment commentA = CommentDAO.storeComment(RATING_A, DRINK_A, CONTENT_A, STORE_A, EMAIL_A);
+    Comment commentA =
+        CommentDAO.storeComment(RATING_A, DRINK_A, CONTENT_A, STORE_A, EMAIL_A, null);
```
Comment (albert-g3) [inline src/test/java/com/google/sps/data/CommentDAOTest.java:59]: instead of null it should be realistic value, but ok for now

**Review by albert-g3 — APPROVED** [review]

File: src/main/webapp/script.js (hunk 3 above)
Comment (TylerHernandez) [reply src/main/webapp/script.js:58]: https://screenshot.googleplex.com/8h0pGxskcr5

**Review by dag13 — APPROVED** [review]

**Pull request merged**